game-core/src/test/java/games/strategy/triplea/delegate/MockDelegateBridge.java

```diff
@@ -58,7 +59,7 @@ public static IDelegateBridge newDelegateBridge(final GamePlayer gamePlayer) {
   public static Answer<int[]> withValues(final int... values) {
     return invocation -> {
       final int count = invocation.getArgument(1);
-      assertEquals(values.length, count, "count of requested random values does not match");
+      assertThat("count of requested random values does not match", values.length, is(count));
```

Comment:
I modified this because the order of elements was wrong.  It is `(expected, actual, description)` and it is being used as `(actual, expected, description)`.  I switched to assertThat while modifying this.

Reply:
Minor, there is a `IsArrayWithSize.arrayWithSize()` matcher that can be used here.

Side-note/observation: It's really odd for there to be an assertion in mock code to begin with :thinking:

Reply:
That doesn't work with non-object types.  I'd have to convert `values` from an `int[]` to an `Integer[]`.